python/across_api/_version.py

```diff
@@ -0,0 +1,17 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```

Comment:
We aren't currently tagging releases, so I'm not sure that setuptools_scm will be very functional. Can we remove this?

Reply:
Yes, I'll remove it.

Reply:
Check in  258bfae fixes this.

Reply:
[52c5fd2](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1742/commits/52c5fd2704cc0a05e8780c28cdcc0a72137f5bb6)